frontend/src/container/NewWidget/RightContainer/Threshold/Threshold.styles.scss

```diff
@@ -297,6 +297,16 @@
 				box-shadow: none;
 			}
 		}
+
+		.invalid-unit {
+			color: var(--bg-vanilla-400);
+			font-family: 'Space Mono';
```

Comment:
We use Giest Mono now, Please update this accordingly.

Reply:
Done